src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -115,6 +115,17 @@
   @end
 @end
 
+@private paramDefault(param)
```

Comment:
The default value should be what's specified in the config file. For example, `edition` defaults to `123`.

Were you planning to do this in a subsequent PR? If so, add a `# FIXME` here

Reply:
I didn't realize that's what we want. That's easy.